_posts/2015-09-05-neo-hpsrt-jekyll-template.markdown

```diff
@@ -0,0 +1,28 @@
+---
+layout: post
+title: Neo-HPSTR Jekyll Blog template
+date: 2015-09-05 12:00:00
+homepage: http://github.com/aron-bordin/neo-hpstr-jekyll-theme
+download: http://github.com/aron-bordin/neo-hpstr-jekyll-theme/archive/master.zip
+demo: http://aron-bordin.github.io/neo-hpstr-jekyll-theme/
+author: Aron Bordin
+thumbnail: neo-hpstr.png
+license: MIT
+license_link: https://raw.githubusercontent.com/aron-bordin/neo-hpstr-jekyll-theme/master/LICENSE
+---
+### Neo-HPSTR Jekyll Template
```

Comment:
Please add an empty line after front mater declaration and add empty lines after each heading.